### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by farishkash** [holyoliver/colmaracademy#5]
Overall met expectations.

Visually the site met the requirements and matched the spec given.

The main issue is that better organizational structure needs to be put in place.

The rest are minor issues such as alt text, but that should be done also.

Now for your consideration, if you haven't already reviewed the Codecademy CSS grid course please do so. You can also go to http://cssgridgarden.com/ for additional practice. As a challenge consider converting this project and incorporating cssgrid into this.
